src/commands/synthetics/utils/internal.ts

```diff
@@ -87,3 +88,98 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
 
   return undefined
 }
+
+type validTestOverridesValues = 'boolean' | 'number' | 'string' | 'string[]' | 'ExecutionRule'
+type AccumulatorBaseConfigOverride = Omit<
+  BaseConfigOverride,
+  'basicAuth' | 'headers' | 'cookies' | 'deviceIds' | 'locations' | 'tunnel' | 'variables' | 'retry'
```

Comment:
Maybe just add a comment here explaining why we're excluding these and adding information about the tickets that describe the work of them being added later on